web/src/sessions.rs

```diff
@@ -47,7 +47,7 @@ impl Session {
             cja::tower_cookies::Cookie::build(("session_id", session.session_id.to_string()))
                 .path("/")
                 .http_only(true)
-                .secure(true)
+                .secure(std::env::var("HTTPS").unwrap_or_else(|_| "true".to_string()) == "true")
```

Comment:
Safari doesn't allow secure cookies on localhost. Most other browsers treat localhost as a 'secure' set and allow you to use secure cookies like you could on a `HTTPS` site.

Safari wasn't in the testing, so this creates a way for testing to NOT set a secure cookie.

This is kinda annoying... But its easier than getting CI to run with a nice https certificate right now!